### ISSUE TITLE
Set system-level file permissions to read-only for a frozen dataset...

### ISSUE THREAD

**Issue by jotelha** [jic-dtool/dtoolcore#22]
... in DiskStoragebroker's https://github.com/jic-dtool/dtoolcore/blob/065b478fd49ffaab5969f493a175200998f69f03/dtoolcore/storagebroker.py#L754 if possible. This suggestion came up today in an introduction to dtool for a few interested PIs here in Freiburg.

**Comment by tjelvar-olsson** [comment]
I don't think that is the responsibility of dtool and it would make the tool very fragile as it would depend on the user invoking dtool having relevant permissions to change the files permissions.
